charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -198,7 +198,6 @@ spec:
 {{- end }}
   podTemplateSpec:
     spec:
-      securityContext: {{- toYaml .Values.cassandra.podSecurityContext | nindent 8 }}
```

Comment:
Do we want to consider removing the entry in our [values.yaml](https://github.com/adejanovski/k8ssandra/blob/f293c5df39957f3e9a0cbe816e65c6832bf9a5ba/charts/k8ssandra/values.yaml#L46) that corresponds to this pod level security for cassdc?  This may avoid confusion for those expecting it to be applied in this version.

Reply:
I agree that it could help avoid confusion to remove the property in values.yaml.